sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -250,14 +250,37 @@ function parseEntryResult(entry: any): object | undefined {
   return undefined;
 }
 
+/**
+ * @internal
+ * @ignore
+ * Utility to help parse link info from the given `feed` result
+ * @param feedLink
+ */
+function parseLinkInfo(
+  feedLink: { [Constants.XML_METADATA_MARKER]: { rel: string; href: string } }[],
+  relationship: "self" | "next"
+): string | undefined {
+  if (!feedLink) {
```

Comment:
```suggestion
  if (!feedLink || !Array.isArray(feedLink) {
```

could do all your error checking here

Reply:
Thanks, Updated!